Exception when running the CLI if the configuration folder exists but the configuration file doesn't

Steps to reproduce:
- Remove configuration file `rm ~/.op/config.yml`
- Run the tool `go run main.go -h`

Expected behaviour: the config file is created
Actual behaviour: 
```
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/elastic/metricbeat-tests-poc/cli/config.checkServices(0x0, 0x0)
        /Users/mdelapenya/sourcecode/src/github.com/elastic/metricbeat-tests-poc/cli/config/config.go:362 +0x1b2
github.com/elastic/metricbeat-tests-poc/cli/config.readConfig(0xc000243dc0, 0x15, 0x11d747b, 0x1b64d20, 0x1c90460, 0x0)
        /Users/mdelapenya/sourcecode/src/github.com/elastic/metricbeat-tests-poc/cli/config/config.go:444 +0x190
github.com/elastic/metricbeat-tests-poc/cli/config.newConfig(0xc000243dc0, 0x15)
        /Users/mdelapenya/sourcecode/src/github.com/elastic/metricbeat-tests-poc/cli/config/config.go:303 +0x78
github.com/elastic/metricbeat-tests-poc/cli/config.InitConfig()
        /Users/mdelapenya/sourcecode/src/github.com/elastic/metricbeat-tests-poc/cli/config/config.go:266 +0xab
github.com/elastic/metricbeat-tests-poc/cli/cmd.init.0()
        /Users/mdelapenya/sourcecode/src/github.com/elastic/metricbeat-tests-poc/cli/cmd/run.go:16 +0x37
exit status 2
```

--- a/cli/config/config.go
+++ b/cli/config/config.go
@@ -304,7 +304,7 @@ func newConfig(workspace string) {
 }
 
 func checkConfigFile(workspace string) {
-	found, err := exists(workspace)
+	found, err := exists(path.Join(workspace, fileName))
 	if found && err == nil {
 		return
 	}

--- a/cli/services/service.go
+++ b/cli/services/service.go
@@ -243,7 +243,7 @@ func (s *DockerService) Run() (testcontainers.Container, error) {
 		ExposedPorts: exposedPorts,
 		Labels:       s.Labels,
 		Name:         s.ContainerName,
-		SkipReaper:   !s.Daemon,
+		SkipReaper:   s.Daemon,
 		WaitingFor:   s.WaitFor,
 	}
 

--- a/metricbeat-tests/.gitignore
+++ b/metricbeat-tests/.gitignore
@@ -7,3 +7,4 @@ outputs/*
 .github/releases
 
 *.swp
+op

--- a/metricbeat-tests/runner_test.go
+++ b/metricbeat-tests/runner_test.go
@@ -180,11 +180,6 @@ func thereAreNoErrorsInTheIndex(index string) error {
 							"event.module": query.EventModule,
 						},
 					},
-					{
-						"match": map[string]interface{}{
-							"service.version": query.ServiceVersion,
-						},
-					},
 				},
 			},
 		},

